CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `SeccmopError::raw_ffi_value`
 - Example code for userspace notification functions.
 - `ScmpFilterContext::get_api_tskip` and `ScmpFilterContext::set_api_tskip`
+- Support for `2.5.5`, `2.5.6`, limited support for `2.6.0`
+  - `.5/.6` support is only fully true if we add the new syscalls to `const-syscalls`
```

Comment:
```suggestion
```

Reply:
We should add the new syscalls, possible in a follow up PR.

Reply:
thanks!